cluster/images/obs-csi-plugin/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM centos:7.6.1810
+
+LABEL maintainers="HuaweiCloud Authors"
+LABEL description="HuaweiCloud CSI Plugin"
+
+ARG centosName=obsfs_CentOS7.6_amd64.tar.gz
+ARG ubuntuName=obsfs_Ubuntu16.04_amd64.tar.gz
+RUN curl https://obs-community.obs.cn-north-1.myhuaweicloud.com/obsfs/current/${centosName} -o /root/${centosName}
+RUN curl https://obs-community.obs.cn-north-1.myhuaweicloud.com/obsfs/current/${ubuntuName} -o /root/${ubuntuName}
+
+COPY entrypoint.sh /entrypoint.sh
+COPY socket-server /bin/socket-server
```

Comment:
`cluster/images/obs-csi-plugin/socket-server` is not required.
Normally, `socket-server` is built in a `Dockerfile` and then copied.

Reply:
ok, cluster/images/obs-csi-plugin/socket-server has been removed.
socket-server will built in Makefile with command :build-socket-server